in_toto/runlib.py

```diff
@@ -47,13 +48,15 @@
   try:
     import subprocess32 as subprocess
   except ImportError:
-    log.warn("POSIX users (Linux, BSD, etc.) are strongly encouraged to"
-        " install and use the much more recent subprocess32")
+    in_toto.settings.COLOR = in_toto.util.detect_colorization(sys.argv)
```

Comment:
Please don't use `sys.arg` in the library module. If a user wants colorized output s/he should set it to True in the command line tool or by directly modifying the `in_toto.settings.COLOR`.

Reply:
Sorry about that. That's been removed now.